city-infrastructure-platform/urls.py

```diff
@@ -1,9 +1,21 @@
 from django.conf import settings
 from django.contrib import admin
-from django.urls import path
+from django.urls import include, path
+from rest_framework import routers
+
+from traffic_control import api as traffic_control_api
+from users import api as users_api
+
+router = routers.DefaultRouter()
+router.register("users", users_api.UserViewSet)
+router.register("groups", users_api.GroupViewSet)
```

Comment:
Maybe I missed something, but could you clarify a bit the need to expose users and groups in REST APIs? Exposing user/group info (especially the whole list of them) may have potential security implications.

Reply:
We definetly need the user info through REST API at some point, but let's skip this for now, because the requirements are not fully known yet and it is directly not part of this PR.